astro/src/content/docs/apis/_application-response-body-base.mdx

```diff
@@ -104,6 +104,14 @@ import Xmlsignaturec14nmethodValues from 'src/content/docs/_shared/_xmlSignature
     * `SlidingWindowWithMaximumLifetime` - the expiration is calculated from the last time the token was used, or until the <InlineField>maximumTimeToLiveInMinutes</InlineField> is reached. <span class="text-green-600">Available since 1.46.0</span>
   </APIField>
 
+  <APIField name={ props.base_field_name + ".jwtConfiguration.refreshTokenOneTimeUseConfiguration.gracePeriodInSeconds" } type="Integer" since="1.55.0">
```

Comment:
```suggestion
  <APIField name={ props.base_field_name + ".jwtConfiguration.refreshTokenOneTimeUseConfiguration.gracePeriodInSeconds" } type="Integer" since="1.55.1">
```